Assets/_BForBoss/_Leaderboard/Scripts/InputUsernameViewBehaviour.cs

```diff
@@ -32,6 +33,7 @@ public void Initialize(InputUsernameViewModel viewModel)
             _saveUsernameButton.onClick.AddListener(() =>
             {
                 _viewModel.SetUserName(_usernameField.text);
```

Comment:
I'm wondering can you `MixPaanel.setUserName` within the viewModel. since viewModels are more reused than View's